x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/category_stopped_partitions.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useContext, useEffect, useState, useMemo } from 'react';
+import { EuiBasicTable, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { JobCreatorContext } from '../../../job_creator_context';
+import { CategorizationJobCreator } from '../../../../../common/job_creator';
+import { Results } from '../../../../../common/results_loader';
+import { ml } from '../../../../../../../services/ml_api_service';
+
+const NUMBER_OF_PREVIEW = 5;
+export const CategoryStoppedPartitions: FC = () => {
+  const { jobCreator: jc, resultsLoader } = useContext(JobCreatorContext);
+  const jobCreator = jc as CategorizationJobCreator;
+  const [tableRow, setTableRow] = useState<Array<{ count?: number; example: string }>>([]);
+  const [hasStoppedPartitions, setHasStoppedPartitions] = useState(false);
+
+  let _resultsSubscription;
+
+  const columns = useMemo(
+    () => [
+      {
+        field: 'partitionName',
+        name: i18n.translate(
+          'xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsPreviewColumnName',
+          {
+            defaultMessage: 'Stopped partition name',
```

Comment:
I'd add in i18n handling for the plural use case here `Stopped partition names` if more than 1 stopped partition.

Reply:
Added here https://github.com/elastic/kibana/pull/75709/commits/618fa8fab2d16c4406f007e3d8c367b7c1ad5795

Reply:
I only have one partition listed here, but it says `names`. But having seen how this looks, I actually think it's fine to go with a heading of `Stopped partition names` for all cases:

![image](https://user-images.githubusercontent.com/7405507/91155302-8ea98000-e6ba-11ea-910d-815038e554e0.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/75709/commits/676dfff60738922e0ffdb7fa6c192fcc588c9a0d